lib/puppet/parser/compiler.rb

```diff
@@ -485,7 +486,7 @@ def evaluate_ast_node
     end
 
     unless (astnode ||= krt.node("default"))
-      raise Puppet::ParseError, "Could not find node statement with name 'default' or '#{node.names.join(", ")}'"
+      raise Puppet::ParseError, _("Could not find node statement with name 'default' or '%{names}'") % { names: node.names.join(", ") }
```

Comment:
Do we need to translate `'default'` ? Will users in different languages be using the word 'default' in their own language?